go/response_handlers.go

```diff
@@ -1465,13 +1579,65 @@ func handleStringToAnyMapResponse(response *C.struct_CommandResponse) (map[strin
 	if typeErr != nil {
 		return nil, typeErr
 	}
+
 	result, err := parseMap(response)
 	if err != nil {
 		return nil, err
 	}
 	return result.(map[string]any), nil
 }
 
+func handleLCSMatchResponse(response *C.struct_CommandResponse, lcsResponseType models.LCSResponseType) (*models.LCSMatch, error) {
+
+	switch lcsResponseType {
+	case models.SimpleLCSString:
+		lcsResp, err := handleStringResponse(response)
+		if err != nil {
+			return nil, err
+		}
+		return &models.LCSMatch{
+			MatchString: lcsResp,
+			Matches:     make([]models.LCSMatchedPosition, 0),
+			Len:         0,
```

Comment:
If you keep it there, cosider renaming to `TotalLen` or so - it holds the total (sum) lengths of all matches.

Reply:
Ok